dll/win32/syssetup/wizard.c

```diff
@@ -411,6 +414,43 @@ static const PRODUCT_OPTION_DATA s_ProductOptionData[] =
     { L"\0", L"WinNT", 1, 0x300, 1 }
 };
 
+// FIXME: uncomment appropriate sound events once they are implemented
```

Comment:
They all can be uncommented here, just check for actual file existence in your function

Reply:
Done.